# HG changeset patch
# User oleh-derevenko
# Date 1547341082 -7200
# Node ID 3262016ef92b4925253411592270c10899e9c652
# Parent  96154dbbff9437c721130272f6f20584b5ec96d8
Fixed: atomicord32 type signedness fixed for mutex-aided atomics implementation

diff --git a/ou/include/ou/atomic.h b/ou/include/ou/atomic.h
--- a/ou/include/ou/atomic.h
+++ b/ou/include/ou/atomic.h
@@ -1630,7 +1630,7 @@
 BEGIN_NAMESPACE_OU();
 
 
-typedef int32_t atomicord32;
+typedef uint32_t atomicord32;
 typedef void *atomicptr;
 
 
